graphql_client_codegen/src/codegen.rs

```diff
@@ -8,10 +8,29 @@ use query::QueryContext;
 use schema;
 use selection::Selection;
 
+pub(crate) fn select_operation(query: &query::Document, struct_name: &str) -> Option<Operation> {
+    let mut operations: Vec<Operation> = Vec::new();
+
+    for definition in query.definitions.iter() {
+        match definition {
+            query::Definition::Operation(op) => {
+                operations.push(op.into());
+            }
+            _ => (),
+        }
+    }
+
+    operations
+        .iter()
+        .find(|op| op.name == struct_name)
+        .map(|i| i.to_owned())
+        .or_else(|| operations.iter().next().map(|i| i.to_owned()))
+}
```

Comment:
I'll document that before merging, so people don't need to read the implementation to know what it does.